setup.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    remove: [
+        { file: 'resources/icons/512x512.png' },
+        { file: 'resources/osx/dmg-background.png' },
+        { file: 'resources/osx/dmg-background@2x.png' },
+        { file: 'resources/osx/dmg-icon.icns' },
+        { file: 'resources/osx/icon.icns' },
+        { file: 'resources/windows/icon.ico' },
+        { file: 'resources/windows/setup-icon.ico' },
+        { folder: 'src/hello_world' }
```

Comment:
Not sure I understand..?  I don't see any suggested "changes", let me know if I'm missing something! 😄